extensions/standard-processors/tests/unit/ExtractTextTests.cpp

```diff
@@ -162,17 +160,34 @@ TEST_CASE("Test usage of ExtractText in regex mode", "[extracttextRegexTest]") {
     test_file.close();
   }
 
-  plan->runNextProcessor();  // GetFile
-  plan->runNextProcessor();  // ExtractText
-  plan->runNextProcessor();  // LogAttribute
+  SECTION("Do not include capture group 0") {
+    plan->setProperty(maprocessor, org::apache::nifi::minifi::processors::ExtractText::IncludeCaptureGroupZero.getName(), "false");
+
+    testController.runSession(plan);
+
+    std::list<std::string> suffixes = { "", ".0", ".1" };
+    for (const auto& suffix : suffixes) {
+      ss.str("");
+      ss << "key:" << "RegexAttr" << suffix << " value:" << ((suffix == ".1") ? "80" : "130");
```

Comment:
I see that this is old code, but we could rewrite it like you did with the other section, explicitly listing the expected log lines, I found this pretty hard to read, while the new version is more obvious IMO